### ISSUE TITLE
Resolving jce_policy-6.zip failed: Name or service not known

### ISSUE THREAD

**Issue by jab416171** [flexiondotorg/oab-java6#105]
This failed on me on line 519:

```
DOWNLOAD_PATH=`grep "jce[^']*-6-oth-JPR'\]\['path" /tmp/oab-download-jce.html | cut -d'=' -f2 | cut -d'"' -f2`
```

`DOWNLOAD_PATH` is empty, so `DOWNLOAD_URL` is just jce_policy-6.zip

I did `grep "jce" /tmp/oab-download-jce.html` and got no output. If you want I can upload my oab-download-jce.html somewhere.

**Comment by ehsanullahjan** [comment]
I'm also getting exactly the same issue. I can build Java 7 packages but building Java 6 packages fail with the error mentioned.

**Comment by haneefmubarak** [comment]
Duplicate of #101.

**Comment by flexiondotorg** [comment]
Merged pull-requuest #111.
